.gitignore

```diff
@@ -7,3 +7,5 @@ coverage
 build
 .DS_STORE
 lib
+__tests__
+package-lock.json
```

Comment:
We should check-in the `package-lock.json` since [npm recommends it](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json).